skribenten-backend/.nais/nais.yaml

```diff
@@ -74,7 +74,7 @@ spec:
   env:
     {{#each env as |e|}}
       - name: {{e.name}}
-        value: {{{e.value}}}
+        value: "{{{e.value}}}"
```

Comment:
Hva gjør denne endringen?

Reply:
Fikk valideringsfeil på at `true` eller `false` i miljø-yaml blir til boolean, men det forventes string. Så dette bare sørger for at det er en string.